System-Architecture/CSC-Overview/Main-Telescope-CSCs/index.rst

```diff
@@ -1,33 +1,103 @@
 .. This is a template top-level index file for a directory in the procedure's arm of the documentation
 
 .. This is the label that can be used as for cross referencing in the given area
-.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hypens
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens
 .. _CSC-Overview-Main-Telescope-CSCs:
 
 ###################
 Main Telescope CSCs
 ###################
 
-.. note::
-    This is a template file that is associated with a template directory structure.
-    This note will be deleted when the section is properly populated
-
-This is the top-level description of what this section is about and what it includes.
-Below is a table of contents which is used to show the levels below this level.
-If there are no levels beneath this one, make sure the 2nd line is commented out
-
-.. toctree::
-    :maxdepth: 2
-    :titlesonly:
-    :glob:
-
-    *
-..    */index
-
-..  Any Figures should be stored in the same directory as this file.
-    To add images, add the image file (png, svg or jpeg preferred) to the same directory as this .rst file.
-    The reST syntax for adding the image is:
-    .. figure:: /filename.ext
-        :name: fig-label
-        :target: http://target.link/url
-        Caption text.
+.. warning::
+    This page is under heavy development and is subject to change.
+
+As mentioned in the :ref:`Control-Packages-General-CSC-Overview`, it is convenient to group CSCs into one of three categories, one of which is the Main Telescope (or MT).
+All CSCs associated with the Main Telescope receive an ``MT`` prefix to their name, such as the MTMount or MTDome.
```

Comment:
Might worth adding a footnote mentioning that this standard was adopted later on in the project and some CSCs (being developed by external contractors) maintained their original name (with the MT suffix) and that we are working to redact that.